src/cli/self_update.rs

```diff
@@ -166,7 +166,8 @@ To get started you need Cargo's bin directory ({cargo_home}/bin) in your `PATH`
 environment variable. Next time you log in this will be done
 automatically.
 
-To configure your current shell run `source {cargo_home}/env`
+To configure your current shell run :
+source {cargo_home}/env
```

Comment:
One point, it's common to indicate actual shell commands either with a `$` at the start of the line, or by indenting.

I would prefer:

```suggestion
    source {cargo_home}/env
```

Though if others disagree, I'm happy to merge this as-is.

Reply:
Indenting makes sense to me; a $ at the beginning is an antipattern as it makes copy-paste harder.